nitransforms/io/itk.py

```diff
@@ -226,11 +225,12 @@ def from_string(cls, string):
         lines = [l.strip() for l in string.splitlines()
                  if l.strip()]
 
-        if lines[0][0] != '#' or 'Insight Transform File V1.0' not in lines[0]:
-            raise ValueError('Unknown Insight Transform File format.')
+        if not lines or not lines[0].startswith('#') or \
+           'Insight Transform File V1.0' not in lines[0]:
```

Comment:
indentation is too close to following line - i've recently become a big fan of writing multiconditional `if`s in the form of 
```python
if (
   condition1
   and condition2
   ...
):

Reply:
thanks, I like it too! but I prefer to have the logical operators at the end of the line